metrics-dropwizard/src/main/scala/com/mesosphere/usi/metrics/dropwizard/reporters/DataDogUDPReporter.scala

```diff
@@ -41,38 +41,46 @@ class DataDogUDPReporter(settings: DataDogUdpReporterSettings, registry: MetricR
   }
 
   private def report(socket: ActorRef): Unit = {
-    report(
-      socket,
-      registry.getGauges, registry.getCounters, registry.getHistograms, registry.getMeters, registry.getTimers)
+    report(socket, registry.getGauges, registry.getCounters, registry.getHistograms, registry.getMeters, registry.getTimers)
   }
 
-  private def report(
-    socket: ActorRef,
-    gauges: util.SortedMap[String, Gauge[_]],
-    counters: util.SortedMap[String, Counter],
-    histograms: util.SortedMap[String, Histogram],
-    meters: util.SortedMap[String, Meter],
-    timers: util.SortedMap[String, Timer]): Unit = {
+  private def report(socket: ActorRef,
+                     gauges: util.SortedMap[String, Gauge[_]],
+                     counters: util.SortedMap[String, Counter],
+                     histograms: util.SortedMap[String, Histogram],
+                     meters: util.SortedMap[String, Meter],
+                     timers: util.SortedMap[String, Timer]): Unit = {
```

Comment:
See my comment above. I prefer the Marathon way on the left.

Reply:
+1 😬

Reply:
I agree it's different but I find this pretty readable. What's wrong with this?

Reply:
It kind of wastes real estate. If everyone else is fine let's use this. However, Marathon has a different config. That's confusing.

Reply:
agree... we can't limit to 120 AND indent 60 :)

Reply:
Fair point, that I can relate to when it's not "I don't like this". I'll change it, thanks